__tests__/scss-valid.scss

```diff
@@ -1,3 +1,151 @@
-@use 'sass:math';
+@use "base";
+@use "sass:list";
+@use "sass:map";
+@use "sass:math";
+@use "sass:meta";
+@use "sass:string";
 
-@import 'bootstrap';
+// Multi-line double-slash comment
+// comment
+
+@function remove-where($list, $condition) {
+  $new-list: ();
+  $separator: list.separator($list);
+
+  @each $element in $list {
+    @if not meta.call($condition, $element) {
+      $new-list: list.append($new-list, $element, $separator: $separator);
+    }
+  }
+
+  @return $new-list;
+}
+
+@mixin corner-icon($name, $top-or-bottom, $left-or-right) {
+  .icon-#{$name} {
+    background-image: url("/icons/#{$name}.svg");
+    position: absolute;
+    #{$top-or-bottom}: 0;
+    #{$left-or-right}: 0;
+  }
+}
+
+@mixin triangle($size, $color, $direction) {
+  border-color: transparent;
+  border-style: solid;
+  border-width: math.div($size, 2);
+
+  @if $direction == up {
+    border-bottom-color: $color;
+  } @else if $direction == right {
+    border-left-color: $color;
+  } @else if $direction == down {
+    border-top-color: $color;
+  } @else if $direction == left {
+    border-right-color: $color;
+  } @else {
+    @error "Unknown direction #{$direction}.";
+  }
+}
+
+@mixin theme($theme: DarkGray) {
+  background: $theme;
+  box-shadow: 0 0 1px rgb($theme / 25%);
+}
+
+%message-shared {
+  border: 1px solid #ccc;
+  color: #333;
+}
+
+$local-color: #333;
+$fonts: "Tahoma", "Helvetica Neue", "Helvetica", sans-serif;
+$theme-colors: (
+  "success": #28a745,
+  "warning": #ffc107,
+);
+
+:root {
+  --brand-red: hsl(5deg 10% 40%);
+}
+
+.selector-1,
+.selector-3[type="text"] {
+  background: linear-gradient(#fff, rgb(0 0 0 / 80%));
+  color: base.$primary-color;
+  top: calc(100% - 2rem) !important;
+
+  a {
+    width: math.div(300px, 960px) * 100%;
+  }
+}
+
+.selector-x { width: 10%; }
+.selector-y { width: 20%; }
+.selector-z { width: 30%; }
+
+.selector-a {
+  @include theme;
+}
+
+.selector-b {
+  @extend %message-shared;
+  @include theme($theme: DarkRed);
+}
+
+/* Single-line comment */
+
+@media (min-width >= 60em) {
```

Comment:
Isn't a `>=` in a media query invalid CSS?

Reply:
It's [range context](https://www.w3.org/TR/mediaqueries-4/#range-context).

I'll take it out though to keep the CSS simple as the emphasis in the fixture should be on SCSS.